refactor: save the code with `ctrl + s`

Alfred-Skyblue opened the pull request:
If you use the change event to save the code, every input triggers the save, I refactored it to use `ctrl+s` to save the code, and if there is no input within five seconds, the code is automatically saved.

sxzz reviewed (changes requested):
Maybe we can have an option for auto saving.

Alfred-Skyblue commented:
> Maybe we can have an option for auto saving.

When the `onChange` event triggers frequently, it can be disruptive, especially when I haven't finished coding. To address this issue, I've implemented a debounce and added an `autoSave` attribute to control the frequency of automatic saving.

sxzz commented:
Done by #247
